.github/workflows/Makefile

```diff
@@ -0,0 +1,19 @@
+check: github-workflow.json-schema yajsv yq
+	for f in *.yml; do ./yq -o json "$$f" > "$${f%yml}json"; done
```

Comment:
super nit, but now that we'll potentially have a lot of `yaml` (but no `yml`) files in the repo with the config changes should we just decide on one supported extension? I suppose we can say configs will be `yaml` and GH manifests will be `yml`

Reply:
Yeah, this is just for the github actions, as it seems like it all prefers `yml`.